### PR TITLE
Use fuzzer variants for run-time flags only.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Remove complex and unmaintainable build argument support.
For fuzzers needing different build time variants should
be just use a different fuzzer directory e.g.
"fuzzer_build_config_1"

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Just a sec, i might have to bring UNDERLYING_FUZZER back

**Comment by inferno-chromium** [comment]
Not ready for review, need to fix generate_makefile.py

File: experiment/run_experiment.py [hunk 1]
```diff
@@ -157,21 +157,16 @@ def validate_fuzzer(fuzzer: str):
 
 def validate_fuzzer_config(fuzzer_config):
```
Comment (jonathanmetzman) [inline experiment/run_experiment.py:158]: I really want to get rid of command line fuzzer variants: https://github.com/google/fuzzbench/issues/292

Comment (inferno-chromium) [reply experiment/run_experiment.py:158]: Will remove tomorrow.

**Pull request merged**